docs/serving/samples/multi-container/README.md

```diff
@@ -0,0 +1,259 @@
+A simple web app written in Go that you can use for multi container testing.
+
+## Prerequisites
+
+- A Kubernetes cluster with Knative installed and DNS configured. Follow the
+  [installation instructions](../../../install/README.md) if you need to
+  create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+- Make sure multi-container flag is enabled as part of `config-features` configmap.
+
+The following steps show how you can use the sample code and deploy the app to your
+cluster.
+
+You can download a working copy of the sample, by entering the
+following command:
+
+```shell
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+```
+
+## Using the sample code
+
+To test multi container functionality, you must create two containers: a serving container, and a sidecar container.
+
+The `multi-container` directory is provided in the sample code, and contains predefined code and dockerfiles for creating the containers.
+
+You can update the default files and YAML by using the steps outlined in this section.
+
+### Serving Container
+1. After you have cloned the sample repository, navigate to the servingcontainer directory:
+
+   `cd knative-docs/docs/serving/samples/multi-container/servingcontainer`
+1. Create a basic web server which listens on port 8881.
+You can do this by copying the following code into the `servingcontainer.go` file:
+
+   ```go
+   package main   
+   import (
+   	"fmt"
+   	"io/ioutil"
+   	"log"
+   	"net/http"
+   )
+   func handler(w http.ResponseWriter, r *http.Request) {
+   	log.Println("serving container received a request.")
+   	res, err := http.Get("http://127.0.0.1:8882")
+   	if err != nil {
+   		log.Fatal(err)
+   	}
+   	resp, err := ioutil.ReadAll(res.Body)
+   	if err != nil {
+   		log.Fatal(err)
+   	}
+   	fmt.Fprintln(w, string(resp))
+   }
+   func main() {
+   	log.Print("serving container started...")
+   	http.HandleFunc("/", handler)
+   	log.Fatal(http.ListenAndServe(":8881", nil))
+   }
```

Comment:
Is there a reason to duplicate the code & Dockerfiles in the README.md?

Reply:
I think it's fine if you don't and just leave the commands that explain how to build and run the multiple container knative service

Reply:
Just added to show it clearly for both serving and sidecar container